generator/main.ts

```diff
@@ -14,20 +14,18 @@
  * limitations under the License.
  */
 
+import * as changeCase from 'change-case'
 import * as mkdirp from 'mkdirp';
 import * as nunjucks from 'nunjucks';
 import * as path from 'path';
 
 import * as file from './base/file';
-import * as reader from './reader/simple_reader';
-
-import snakeCase = require('snake-case');
-
 import EnumTypes from './parser/enum_types';
 import IDLDefinition from './parser/idl_definition';
 import IDLEnum from './parser/idl_enum';
 import IDLInterface from './parser/idl_interface';
 import Parser from './parser/parser';
+import * as reader from './reader/simple_reader';
```

Comment:
i don't want to move this line.
but our format command moved this line.

Reply:
Ack; The clang will sort the module string(e.g. ./reader/simple_reader) order.

Reply:
okay i see.